doc/changelog.rst

```diff
@@ -10,25 +10,39 @@ v2.12.0
 -------
 
 Highlights
+~~~~~~~~~~
+
+-  Preprocessor :func:`esmvalcore.preprocessor.extract_time` now allows to
+   extract time blocks in each year by making parameters ``start_year`` and
+   ``end_year`` optional.
+-  A new way of configuring the tool has been developed.
+-  Performance improvements:
+
+    -  An iris-esmf-regrid scheme has been added to
+       preprocessor :func:`esmvalcore.preprocessor.regrid`, which improves
+       the regridding of 2D grids and adds the capability to regrid UGRID meshes
+       out of the box.
+    -  Data is now saved from one preprocessing task at the time when using the
+       distributed scheduler, in order to avoid running out of memory.
+    -  A better default ``num_workers`` has been set when using more than one
+       ``max_parallel_tasks`` with an unconfigured threaded scheduler.
 
-Preprocessor :func:`esmvalcore.preprocessor.extract_time` now allows to
-extract time blocks in each year by making parameters ``start_year`` and
-``end_year`` optional.
+-  An on-the-fly cmorizer for ACCESS native data is now available.
 
 This release includes
 
 Backwards incompatible changes
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 -  Make derivation of total column ozone (`toz`) more flexible and add derivation of stratospheric and tropospheric column ozone (:pull:`2509`) by :user:`schlunma`
--  Merge configuration object from multiple files (instead of one single file) (:pull:`2448`) by :user:`schlunma`
 -  Remove deprecated CMOR fix/check code (:pull:`2552`) by :user:`schlunma`
 -  Remove deprecated statistical operators (:pull:`2553`) by :user:`schlunma`
 -  Save all files in a task at the same time to avoid recomputing intermediate results (:pull:`2522`) by :user:`bouweandela`
```

Comment:
```suggestion
-  Save all files in a task at the same time to avoid recomputing intermediate results (:pull:`2522`) by :user:`bouweandela`
   - The signature of the preprocessor function :func:`~esmvalcore.preprocessor.save`
     has changed. The function now accepts a ``compute`` argument that can be :obj:`True`,
     in which case the return value will be :obj:`None` or :obj:`False`, in which case
     the return value will be a :class:`~dask.delayed.Delayed` object that can be used
     to compute and save the data of the cube.
```